commitlint.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+  rules: {
+    'scope-enum': [2, 'always', ['deps', 'dev-deps', 'configurations', 'error-handler']],
```

Comment:
Consider making this a warning or adding another scope in case we want to add a feature or improve README

Reply:
Oopsi, my bad